tools/Makefile.shared

```diff
@@ -12,10 +12,62 @@
 #*   special exception on linking described in the file LICENSE.          *
 #*                                                                        *
 #**************************************************************************
-
+MAKEFLAGS := -r -R
```

Comment:
Then I have another question: why do you need to remove the default rules and variables, and what does it have to do with installing the native version of the tools?

BTW, on OS X `make` is still version 3.81 of GNU Make.